internal/runners/packages/import.go

```diff
@@ -122,7 +122,7 @@ func (i *Import) Run(params *ImportRunParams) error {
 
 	changeset, err := fetchImportChangeset(reqsimport.Init(), params.FileName, lang.Name)
 	if err != nil {
-		return locale.WrapError(err, "err_obtaining_change_request", "Could not process change set: {{.V0}}.", api.ErrorMessageFromPayload(err))
+		return err
```

Comment:
We should still wrap this error. We can come up with a nicer error message than what was here before or use `errs.Wrap` if you don't want it to be included in the default state tool output.

Reply:
That's a fair point. Thanks for reminding me about `errs.Wrap`. I did want to avoid showing this error to the user.